src/core/evaluator.js

```diff
@@ -1415,9 +1415,7 @@ var PartialEvaluator = (function PartialEvaluatorClosure() {
 
           var glyphUnicode = glyph.unicode;
           var NormalizedUnicodes = getNormalizedUnicodes();
-          if (NormalizedUnicodes[glyphUnicode] !== undefined) {
-            glyphUnicode = NormalizedUnicodes[glyphUnicode];
-          }
+          glyphUnicode = keepUnifiedIdeograph(glyphUnicode, NormalizedUnicodes);
```

Comment:
It doesn't look like `NormalizedUnicodes` is used elsewhere in the file, so how about, instead of passing it as a parameter, removing `NormalizedUnicodes` and its import here and just call it in the other file where it's also defined?

Reply:
1. `NormalizedUnicodes` is generated by function `getNormalizedUnicodes` in the file `core/unicode.js`.  
2. The function `getNormalizedUnicodes` is also used in the file `test/unit/unicode_spec.js`
3. If I remove the `NormalizedUnicodes `, I must change the `test/unit/unicode_spec.js`. So I decide to leave it to the lib designer
4. Also, current way is good for function separation